backend/utils/schemas.py

```diff
@@ -100,5 +102,7 @@ def json_schema_to_pydantic_type(json_schema: dict[str, Any]) -> Any:
             return dict
     elif type_ == "null":
         return Optional[Any]  # Use Optional[Any] for nullable fields
+    elif type_ is "literal":
```

Comment:
should this be
` elif type_ == "literal":`?

Reply:
Thank you for the review. I fixed it.